src/mod_push_service_mongoosepush.erl

```diff
@@ -51,8 +51,9 @@ start(Host, Opts) ->
 
     MaxHTTPConnections = gen_mod:get_opt(max_http_connections, Opts, 100),
     mongoose_wpool:ensure_started(),
-    {ok, _} = mongoose_wpool:start(?MODULE, Host, [{strategy, available_worker},
-                                                   {workers, MaxHTTPConnections}]),
+    {ok, _} = mongoose_wpool:start(generic, Host, mod_push,
```

Comment:
I kind of dislike replacing `?MODULE` with tags that are meant to reference the module but in a subtler way. If we see a pool process with e.g. `pusher_push` in its name, we'll grep around the code to find where it comes from. If the tag was simply `?MODULE`, everything would be clear from the getgo. In this instance, `mod_push` does not really correspond to a pool for `mongoosepush` service.

Reply:
1. Previously `?MODULE` was used as the pool's type which doesn't fit nicely to the convention introduced and explained in #1959.
2. On one hand using ?MODULE as a tag name would be handy, on the other hand in the future the tag name is supposed to be configurable, which makes greping even more difficult.
3. I'm ok with using `?MODULE` as a tag for pool which are not started via `outgoing_pools` option.

@fenek I think your input on this is needed.

Reply:
I'd be fine with merging this as-is if we only change the `mod_push` tag to something a little more fitting.